control-plane/stor-port/src/types/v0/transport/replica.rs

```diff
@@ -74,6 +74,102 @@ impl Replica {
     }
 }
 
+/// The request type to create snapshot of a replica.
+pub struct CreateReplicaSnapshot {
+    /// uuid of the replica.
+    pub replica: ReplicaId,
```

Comment:
Please use capital letters for doc comments to begin.

Reply:
Same for others.

Reply:
fixed.